src/machine/uart.go

```diff
@@ -99,8 +100,17 @@ func (uart *UART) Buffered() int {
 	return int(uart.Buffer.Used())
 }
 
+func (uart *UART) SetRXCallback(cb func(byte) bool) {
+	uart.rxCallback = cb
+}
```

Comment:
This needs documentation that includes the following:
- it is an interrupt (not just a regular callback)
- it should describe what the return value is

See for example:

https://github.com/tinygo-org/tinygo/blob/731532cd2b6353b60b443343b51296ec0fafae09/src/machine/machine_nrf.go#L73-L80